pilot/pkg/config/kube/gateway/testdata/deployment/manual-sa.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
+  annotations: {}
```

Comment:
does this mess with server-side apply ownership at all? Probably not

Reply:
I don't think so; I believe the templates that generate this test yaml exclude the SSA annotations